app/styles/main.scss

```diff
@@ -808,18 +372,6 @@ body {
     margin: 0;
     display: block;
     overflow: hidden;
-
-    .modal-dialog {
```

Comment:
.modal-dialog and .modal-content overloads are actually a fix for displaying bootstrap modal full-screen. They are not used directly by app, but nevertheless are in DOM by Bootstrap. Should be preserved if full-screen functionality is needed,